lib/parallel_cucumber/formatters/json_formatter.js

```diff
@@ -27,12 +28,27 @@ var JsonFormatter = function(options) {
       Debug('New feature');
       self._features.push(currentFeature);
     }
+
+    if (self.outFilePath) {
```

Comment:
@simondean Yes, this unique solution was needed because the continuous output reporting can only be done for json output. In this case, we can update the json file after every scenario, while in case of other formatters like console, this is not possible. So that is why I used this approach.